superset-frontend/src/components/FilterableTable/index.tsx

```diff
@@ -74,7 +65,6 @@ function renderBigIntStrToNumber(value: string | number) {
   return <>{convertBigIntStrToNumber(value)}</>;
 }
 
-const GRID_POSITION_ADJUSTMENT = 4;
 const SCROLL_BAR_HEIGHT = 15;
 // This regex handles all possible number formats in javascript, including ints, floats,
```

Comment:
Can we delete `StyledFilterableTable`?  I'm assuming that the default theme of our table component is sufficient. If not, can you adjust its class references? `.ReactVirtualized_` classes don't even exist in the new component.